From c247d4d20b5dceba8b8f53b520950bdf996da455 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Tue, 30 May 2023 00:06:03 +0200
Subject: [PATCH 406/406] bluetooth: h5: Don't re-initialize rtl8723cs on
 resume

The power is kept across during sleep, and this is not necessary.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/bluetooth/hci_h5.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bluetooth/hci_h5.c b/drivers/bluetooth/hci_h5.c
index e90670955df2..3c30d27b485e 100644
--- a/drivers/bluetooth/hci_h5.c
+++ b/drivers/bluetooth/hci_h5.c
@@ -1103,7 +1103,7 @@ static const struct of_device_id rtl_bluetooth_of_match[] = {
 	{ .compatible = "realtek,rtl8723bs-bt",
 	  .data = (const void *)&h5_data_rtl8723bs },
 	{ .compatible = "realtek,rtl8723cs-bt",
-	  .data = (const void *)&h5_data_rtl8723bs },
+	  .data = (const void *)&h5_data_rtl8822cs },
 	{ .compatible = "realtek,rtl8723ds-bt",
 	  .data = (const void *)&h5_data_rtl8723bs },
 #endif
-- 
2.34.1

